src/test/java/com/salesforce/dynamodbv2/mt/mappers/index/DynamoSecondaryIndexMapperByTypeImplTest.java

```diff
@@ -56,6 +57,7 @@ void testNoMatch() {
             GSI)));
         try {
             new DynamoSecondaryIndexMapperByTypeImpl().lookupPhysicalSecondaryIndex(vsi, physicalTable);
+            fail("It was expected that #lookupPhysicalSecondaryIndex throws MappingException");
```

Comment:
instead, please use "Expected MappingException not thrown" (won't "lookupPhysicalSecondaryIndex" already appear in the stack trace?)—similarly for other changes